src/actors/objects/mod.rs

```diff
@@ -552,29 +596,44 @@ impl ObjectsActor {
         file_metas.and_then(move |responses: Vec<Result<Arc<ObjectFileMeta>, _>>| {
             responses
                 .into_iter()
-                .enumerate()
-                .min_by_key(|(i, response)| {
+                .min_by_key(|response| {
                     // Prefer files that contain an object over unparseable files
-                    let object = match response {
-                        Ok(object) => object,
+                    let object_meta = match response {
+                        Ok(object_meta) => object_meta,
                         Err(e) => {
                             log::debug!("Failed to download: {:#?}", e);
-                            return (3, *i);
```

Comment:
iirc we did this for stable `min` such that ties are resolved by order of sources.

Reply:
The `i` here entirely depends on the order of the futures being executed above above calling `download_svc.list_files()`.  Practically this means there is no order, it's a race on who gets executed by the executor first (for non-sentry sources) or how fast the network is (for the sentry source).

At least that's my understanding and the reason I took it out.  I don't think it enforces anything and only makes the code look more complicated.

Reply:
`join_all` preserves order of futures. `list_files` can return any order it wants, but joining all `list_files` together resembles the source order.

In theory this gives you the possibility to upload your own Windows kernel symbols and let them have precedence over our own built-in symbol source. I am not sure if anybody relies on this.

Reply:
This is definitely behavior we should retain.